src/pages/downloads/_components/HeaderText.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const HeaderText = ({type, text, link}): JSX.Element => {
+
+  const anchor = React.createElement(
+    'a',
+    {className: "hash-link", href: `#${link}`, title: "Direct link to heading"}
+  )
+
+  const heading = React.createElement(
+    type || 'h1',
+    link ? {className: "anchor anchorWithHideOnScrollNavbar_node_modules-@docusaurus-theme-classic-lib-next-theme-Heading-styles-module"} : null,
+    text, link ? anchor : null
+  )
```

Comment:
Everything else looks good but this one isn't fixed yet.

Reply:
Shoot I was rushing and missed those. I'll tag them in the morning.

Reply:
I believe this is fixed now